docs/source/reference-core.rst

```diff
@@ -102,6 +102,8 @@ them. Here are the rules:
     only that one will act as a checkpoint. This is documented
     on a case-by-case basis.
 
+    * :func:`trio.open_nursery` is a further exception to this rule.
```

Comment:
Not sure if this is needed, I think the `Partial exception for async context managers` block handles this

Reply:
It may be overkill, but `open_nursery` in itself as a context manager doesn't guarantee a full checkpoint on either of entry or exit - so imo it doesn't abide by the special rules in the `Partial exception for async context managers` block. There's some more discussion around this in #1457